buildSrc/src/main/java/com/r3/csde/dtos/CpiMetadataDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.r3.csde.dtos;
+
+public class CpiMetadataDTO {
+    // Note, these DTOs don't cover all returned values, just the ones required for CSDE.
+    private String cpiFileChecksum;
+    private CpiIdentifierDTO id;
+
+    public CpiMetadataDTO() {}
+
+
+    public String getCpiFileChecksum() {
```

Comment:
same here  - short 1 liners

In C++ / C# I just used public fields in such cases to save the additional redundant code

Reply:
I don't believe you can do that here, jackson needs the standard getters and setters for serialisation. Its much nicer in kotlin too.